The metaverse explained in 14 minutes by Matthew Ball

HamzaAhmedSheikh opened the pull request:
Matthew Ball explains the metaverse.

HamzaAhmedSheikh committed:
The metaverse explained in 14 minutes by Matthew Ball

Matthew Ball explains the metaverse.

HamzaAhmedSheikh committed:
The world's future superpowers might be Roblox, Minecraft, and Fortnite, according to Matthew Ball.

HamzaAhmedSheikh committed:
how will the Metaverse change my life? by Matthew Ball

HamzaAhmedSheikh committed:
Decentraland announces world’s first Metaverse ATM

Hamzah-syed commented:
Hi Hamza, kindly Include this in the **Resources** section

![image](https://user-images.githubusercontent.com/50793209/195653770-79fed395-b769-412f-b509-47c5cb7de0d9.png)

Hamzah-syed commented:
Include this [Decentraland announces world’s first Metaverse ATM](https://github.com/panaverse/metaversepedia/pull/4/commits/8fec434c946989a5be1d9a3e9463dd6ee75cc8c1) in the **Metaverse News** section.

HamzaAhmedSheikh committed:
Decentraland announces world’s first Metaverse ATM

Decentraland, a popular Metaverse platform, has revealed its intentions to open the first Metaverse ATM.

HamzaAhmedSheikh committed:
Merge branch 'main' into main

Hamzah-syed commented:
Looks good thanks for the contribution
